README.md

```diff
@@ -2,8 +2,52 @@
 
 ## Purpose
 
-_... help fill this in!_
+Purpose is *not* to respond to reports of security issues with Node.js, that is
+done by the "security team", see https://github.com/nodejs/node#security
+
+*todo*: should the team be identified in
+https://github.com/nodejs/node#security?
+
+Purpose is to be discussed in working group meetings, but topics raised in the
+WG meeting at NINA16 included:
+
+- Is it worth looking at getting a sec eval of node?
+
+- What is security? Is there definitions that would help us? does CVE/CERT/?
+  have classifications we can use? It would be helpful to have a stance on this
+  so we can more clearly respond to issues that are reported or claimed as
+  "security issues".
+
+Ecosystem:
+
+- sec-wg could take some responsibility for fixing ecosystem issues
+- Best practices on sec problems could use definition and
+  publication/evanglizing, so npmjs.org packages can follow a well-known and
+  robust procedure:
+  - What should an npm module author do if an issue is found in their module?
+  - How should they deal with it?
+  - Should there be a CVE?
+  - How should we (node/the node foundation/sec-wg) communicate with the
+    author?
+  - Its possible there should be a community vulnerability response list, that
+    is available if people want to report a vulnerability in community modules,
+    but don't know how, and do not want it in a public issue/PR on the modules
+    github repo.
+
+Signed modules:
+- @jasnell has some thoughts on this.
+
+... _anything else I missed?_
+
 
 ## Members
 
 _... could this be you?_
+
+*TODO* @jasnell, @mdawson What is process to "join" a Working Group. Is it PR an
+addition of your name, email address, and github handle to this section of the
+README if you want to be involved?
+
+
+*TODO* Is there some boilerplate we need? Should we describe how meetings are
+announced, for example?
```

Comment:
We need the governance boilerplate, which includes CONTRIBUTING.md, GOVERNANCE.md  they can likely just be copied from one of the other WGs or the TSC or CTC repo.